imageroot/bin/module-dump-state

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+echo "Dumping mattermost postgres database"
+podman exec postgres-app pg_dump -U mattuser --format=c  mattermost > dump.sql
```

Comment:
Just a little thing: the dump format is not SQL, so the .sql extension does not fit. I propose something else, like `.pg_dump` :shrug: 

```suggestion
podman exec postgres-app pg_dump -U mattuser --format=c mattermost > mattermost.pg_dump
```

Reply:
done